### DISCUSSION TITLE
The return value for the "--display-sync-status " parameter.

### DISCUSSION THREAD

**Discussion by leon-zhang-guoao** [abraunegg/onedrive#1567] [Q&A] [answered]
Hi 
I want to know what values the "--display-sync-status "parameter returns and what they mean. This is very important to me, I need to monitor the status of the onedrive.

Thanks

**Comment by abraunegg** [comment, marked as answer]
@leon-zhang-guoao 
> I want to know what values the "--display-sync-status "parameter returns and what they mean. This is very important to me, I need to monitor the status of the onedrive.

I am unsure what you mean here.

If the client is 'in sync' with OneDrive online, the output is:
```
Configuration file successfully loaded
Configuring Global Azure AD Endpoints
Initializing the Synchronization Engine ...
No pending remote changes - in sync
```

If there are changes / differences between OneDrive, the output is as follows:
```
Configuration file successfully loaded
Configuring Global Azure AD Endpoints
Initializing the Synchronization Engine ...
Local directory is out of sync with OneDrive
```

I am unsure how else you can interpret this ... ???


Please can you try and explain better what you are after, what values are you talking about here .. ????

**Reply by leon-zhang-guoao** [reply]
I want to monitor the health.Do you have any good suggestions?

**Reply by abraunegg** [reply]
@leon-zhang-guoao 
> I want to monitor the health.Do you have any good suggestions?

Monitor the health of what?
* The 'onedrive' process itself?
  * Is the onedrive service running
  * Is the onedrive process running
* Is data being transmitted?

Your best bet is it look at the systemd logs or the logging output to monitor what the application is doing.

For anything else, you need to ask your question on the forums that support your chosen Linux distribution.

**Reply by leon-zhang-guoao** [reply]
ok  Thanks
